src/Symfony/Bundle/SecurityBundle/Security/FirewallConfig.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SecurityBundle\Security;
+
+/**
+ * Object representation of a firewall configuration.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class FirewallConfig
+{
+    private $name;
+    private $requestMatcher;
+    private $security;
+    private $provider;
+    private $entryPoint;
+    private $stateless;
+    private $context;
+    private $accessDeniedHandler;
+    private $accessDeniedUrl;
+    private $userChecker;
+    private $listenerConfigs;
+
+    public function __construct($name, $requestMatcher, $security = true, $stateless = false, $provider = null, $context = null, $entryPoint = null, $accessDeniedHandler = null, $accessDeniedUrl = null, $userChecker = null, $listenerConfigs = array())
+    {
+        $this->name = $name;
+        $this->requestMatcher = $requestMatcher;
+        $this->security = $security;
+        $this->stateless = $stateless;
+        $this->provider = $provider;
+        $this->context = $context;
+        $this->entryPoint = $entryPoint;
+        $this->accessDeniedHandler = $accessDeniedHandler;
+        $this->accessDeniedUrl = $accessDeniedUrl;
+        $this->userChecker = $userChecker;
+        $this->listenerConfigs = $listenerConfigs;
+    }
+
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * Returns the request matcher service id.
+     *
+     * @return string
+     */
+    public function getRequestMatcher()
+    {
+        return $this->requestMatcher;
+    }
+
+    /**
+     * Returns either the security is disabled or not.
+     *
+     * @return bool
+     */
+    public function hasSecurity()
```

Comment:
This would sound a bit more like you have something on or off: `isSecurityEnabled()`